components/search/themes/express.css

```
@@ -23,6 +23,9 @@
 		--spectrum-search-border-color-focus-hover: var(--spectrum-gray-900);
 		--spectrum-search-border-color-key-focus: var(--spectrum-gray-900);
 
+		--spectrum-search-background-color-disabled: var(--spectrum-disabled-background-color);
+		--spectrum-search-border-color-disabled: var(--spectrum-disabled-background-color);
```

```suggestion
		--spectrum-search-border-color-disabled: var(--spectrum-disabled-border-color);
```

This might address the Chromatic diff you were seeing.